lib/ts/recipe/session/claimBaseClasses/primitiveClaim.ts

```diff
@@ -30,6 +31,11 @@ export abstract class PrimitiveClaim<T extends JSONValue> extends SessionClaim<T
         return payload[this.key]?.v;
     }
 
+    getLastRefetchTime(payload: any, _userContext?: any): Date | undefined {
```

Comment:
Probably return `number | undefined` - return time in milliseconds? Cause everywhere else in the SDK we use time in milliseconds for everything else.

Reply:
Done.